src/uct/cuda/cuda_ipc/cuda_ipc_iface.h

```diff
@@ -19,36 +19,35 @@
 #define UCT_CUDA_IPC_MAX_PEERS  16
 
 
+typedef struct uct_cuda_ipc_iface_cfg {
```

Comment:
Having `uct_cuda_ipc_iface_cfg` and `uct_cuda_ipc_iface_config` maybe misleading. I would suggest to use `uct_cuda_ipc_iface_config_values`, or  `uct_cuda_ipc_iface_config_params` here. And then:
```C

typedef struct uct_cuda_ipc_iface_config {
    uct_iface_config_t                 super;
    uct_cuda_ipc_iface_config_params_t params;
} uct_cuda_ipc_iface_config_t;
```

Reply:
Done